content/docs/learning-guide/tutorials/physx/wind-provider.md

```diff
@@ -0,0 +1,103 @@
+
+    ![PhysX Wind Configuration tag component setup.](/images/learning-guide/tutorials/physx/wind-tag-setup.png)
+
+    {{< note >}}
+    If you choose to use the **Local wind tag** property, the wind force affects only entities that are inside the volume of the **PhysX Collider** component that you add in the next step.
+    {{< /note >}}
+
+## Define the force region with a PhysX Collider
+
+In this section, you'll add a box shaped PhysX Collider component to the wind provider entity.
+
+1. Add a PhysX Collider component to the wind provider entity. If you are using the **Local wind tag** property, this collider defines the volume that contains the wind force. With the **Global wind tag** property, the size and position of this collider aren't critical because the wind force is global. However, the collider provides a useful visualization for the global wind force.
+
+1. Set the PhysX Collider component's **Shape** property to `Box`.
+
+1. Position and scale the PhysX Collider component. Use the **Move** tool to place the entity in the level. Set the **Box Dimensions** property in the PhysX Collider component as desired. If you are creating a localized wind force, enlarge the collider dimensions to a size that is large enough to contain the entity that receives the wind force. In the following example, the collider is 5 meters in each dimension, and the wind provider entity is positioned so that the bottom of the box is on the ground plane.
+
+    {{< image-width src="/images/learning-guide/tutorials/physx/wind-collider-setup.png" width="900" alt="Setting up a PhysX collider for a force region" >}}
+
+## Create a PhysX Force Region
+
+In this section, you'll set up a **PhysX Force Region** that generates a wind force.
+
+1. Add a PhysX Force Region component to the entity. This component creates the wind force. To configure the component, do the following:
+
+1. Choose the {{< icon "add.svg" >}} **Add** button to add a new force.
+
+1. In the **Direction** property, set the **Y** component a value of `-1.0` and set the **Z** component to `0.0` to create a direction for the wind force.
+
+1. In the **Magnitude** property, set a value of `10.0` to create magnitude for the wind force.
+
+    The PhysX collider box displays cones representing the wind force direction.
+
+    {{< image-width src="/images/learning-guide/tutorials/physx/wind-force-region-setup.png" width="900" alt="Setting up a PhysX collider for a force region." >}}
+
+## Add a Cloth prefab
+
+1. To test the wind provider, add prefab with an NVIDIA Cloth mesh. In **Asset Browser**, navigate to `Gems\NvCloth\Assets\prefabs\Cloth`, locate the `cloth_locked_edge.prefab`, and then **drag** it into the viewport.
+
+1. Use the **Move** tool to place the cloth prefab. If you are using the **Local wind tag** property, you must place the cloth asset inside the PhysX Collider volume of the wind provider entity.
```

Comment:
```suggestion
1. To test the wind provider, add a prefab with an NVIDIA Cloth mesh. In **Asset Browser**, navigate to `Gems\NvCloth\Assets\prefabs\Cloth`, locate `cloth_locked_edge.prefab`, and then drag it into the viewport.

1. Use the Move tool to place the cloth prefab. If you are using the **Local wind tag** property, you must place the cloth asset inside the PhysX Collider volume of the wind provider entity.
```
* Line 67:
  * Add missing word "a" ("add a prefab")
  * Cut "the" before the prefab file name. ("locate [file name]") 
  * Unless "drag" is a UI text label, drop the **bold** formatting.
* Line 69
  * Second occurrence of "Move tool". You can drop the bold formatting for this occurrence.
  * For consistency with the section header ("Add a Cloth prefab"), use either "Cloth prefab" or "cloth prefab" for both occurrences. Your call.